app/views/schemes/support_services_provider.html.erb

```diff
@@ -0,0 +1,36 @@
+<% content_for :title, "Which organisation provides the support services used by this scheme?" %>
+
+<% content_for :before_content do %>
+  <%= govuk_back_link(
+    text: "Back",
+    href: request.query_parameters["check_answers"] ? "/schemes/#{@scheme.id}/check-answers" : "/schemes/#{@scheme.id}/details",
```

Comment:
We have this logic in the view here but in the controller on the other one. Don't mind either approach but should probably be consistent about it?

Reply:
Moved back logic to front, controller is already fat enough